src/agent/debuglet.ts

```diff
@@ -322,6 +329,18 @@ export class Debuglet extends EventEmitter {
 
     });
   }
+  checkReady() {
+    const diff = process.hrtime(this.promiseResolvedTimestamp_);
```

Comment:
Is `process.hrtime` necessary? We only need timestamp in seconds. (Not sure what the best practice is when dealing with timescales on the order of seconds.)

Reply:
+1. process.hrtime() is only for high-resolution time data. We don't need that fine granular data, do we?

Reply:
I used high resolution time here as in legacy-debugapi.ts we have measured capture time using hrtime. I agree that here, measuring time difference at around 30 seconds, we can use Date.now() instead.